yarn.lock

```diff
@@ -16093,6 +16093,11 @@ vue-loader@^15.8.3:
     vue-hot-reload-api "^2.3.0"
     vue-style-loader "^4.1.0"
 
+vue-recaptcha@^1.3.0:
```

Comment:
For the sake of consistency, use the same vue-recaptcha version we use elsewhere (Check EBM website contact-us package) (`1.2.0`), or upgrade both versions. The only major change I see between 1.2 and 1.3 is the addition of language support, and there don't appear to be any backwards incompatible changes.